Packages/com.unity.inputsystem/InputSystem/Devices/Mouse.cs

```diff
@@ -23,7 +23,7 @@ public struct MouseState : IInputStateTypeInfo
         public static FourCC Format => new FourCC('M', 'O', 'U', 'S');
 
         /// <summary>
-        /// Screen-space position of the mouse in pixels.
+        /// Screen-space position of the mouse in pixels. Updated independently from delta using WM. At high framerates, certain frames might not receive any new position data.
```

Comment:
We need to remember this is a platform-agnostic API. I would suggest changing "Updated", which is given, to "May be updated independently from delta depending on the platform."

Then you may add to remarks: "On Windows, delta originates from RAWINPUT API." or leave that out.

Reply:
"If a delta is needed that correlates with position, compute it based on the last reported position".